### DISCUSSION TITLE
Compiling a collection of third party material - Compendium returns blank

### DISCUSSION THREAD

**Discussion by inraptor** [kinkofer/FightClub5eXML#204] [Q&A]
Hey everyone,

Hopefully people are still around. I have been trying to compile a couple xml files I created for some third party material into a Compendium so I can import them all together. The ultimate goal was to add them to the Complete Compendium so its just one import and to ensure everything works together. I have each xml file in the needed format for importing into FightClub. I have test each one and they all work, though the spell list for the third party class seems to be getting overridden by either the complete compendium or the third party spell file (why I'm trying to group everything into one file). I bet I could very easily just copy and paste all the third party material into the complete xml, keeping the organization, but I would like a chance to learn how to use these powerful tools.

I have created a collection file using the same format as the other collection files - "<doc href="../Sources/(Folder for 3rd party content)/(xml file)" />", rinse and repeat a few more times for all of them. I have the version and encoding the same. The doc href are all within the <collection> tag. When I run the execution code to merge, the compendium file is created, but nothing is added to it. Another issue arises with one particular file that produces this error: warning: failed to load external entity.

I have tested the other collection files that came within the download of the github's data and executed them in powershell, they all create a compendium properly. So from what I can tell, I am executing the code correctly. I have verified that the 3rd party xml files all have proper tags and hierarchy's, again, they all work when I import them into FightClub. I'm not entirely sure what I should do to get around this roadblock. Any insights or ideas would be appreciated. I looked at other discussions that were similar and tried adapting what worked there to my situation but no dice (pun not intended but fits for a D&D group).

Thank you for your time,
inraptor

**Comment by abc** [comment]
Hey @inraptor, just to let you know, this repo is unfortunately abandoned and development has moved over to ogre-project/FightClub5eXML for the time being, at least until @kinkofer returns and adds some contributors to this repo. 

I’d be happy to help you out with what you’re trying to do but I think the best way to do that would be for you to share your files. Would you be able to set up a fork of the ogre-project repo with your third party collection files?

**Reply by inraptor** [reply]
Hi @abc. I am very sorry, life got extremely busy and chaotic once I posted my original discussion and am now just seeing this. Thank you for making me aware of the new group. I'd be happy to move over there and share my files. Please be patient with me as life is still a little messy.

**Reply by abc** [reply]
Not a problem at all, I absolutely understand how crazy things can get. I really hope things get better for you soon, take care!
